test/unit_tests/braket/flake8_plugins/example_files/keyword_functions.py

```diff
@@ -7,4 +7,17 @@ def function_0(arg0: str, *, arg1: int = 0, arg2: int = 1, arg3: int = 1) -> int
     Returns:
         int: value of the function.
     """
+    pass
+
+
+def function_1(arg0: str, *, arg1 = 0, arg2: int = 1, arg3: int = 1) -> int:
+    """This is a description.
+    Args:
+        arg0 (str): This is a parameter
+        arg1 (bool): This is missing a type hint
+        arg2 (int): This is another parameter
+        arg3 (int): This is yet another parameter
+    Returns:
+        int: value of the function.
+    """
     pass
```

Comment:
nit: newline. Looks unrelated to this change though. Does this package not get linted by itself or does it not check for newlines?

Reply:
Not for the example files we use for testing. They have errors on purpose.

Reply:
Oh right, that makes sense lol